src/common/bool.js

```diff
@@ -0,0 +1,3 @@
+import Joi from 'joi';
+
+export default description => Joi.boolean().description(description);
```

Comment:
What is this description stuff you are doing here? This should be for just displaying on the api documentation, I don't really understand why you would need to pass that in.